cwd/app.js

```diff
@@ -270,7 +271,7 @@
   function startKiosk(duration) {
     views = allGlyphs.filter(obj => obj.view).map(obj => obj.view);
     let index = 0;
-    let hashes = views.map(getName);
+    hashes = views.map(getName);
```

Comment:
Try to see if you can refactor this so that we don't have to use an ugly global variable.  Maybe it's fine, though, but I would prefer it get defined at the top of the file first or something.